client/app/scripts/liveblog-edit/styles/liveblog-edit.less

```diff
@@ -580,6 +580,13 @@
 }
 
 // preview changes
+
+.close-preview {
+    font-size: 2em;
+    color: #8e8e8e!important;
```

Comment:
I'm not a big fan of `!important` declarations. Mostly outside of a scope.
And you will see that if you put it in a scope, you won't need anymore to use `!important` ;)
Everything in a module style should belong to the module scope, like a common parent class. For global style it's better to put it in `client/app/styles/less/liveblog.less`